Новые ошибки сборки clang'ом

Компилятор clang 14, собираю под FreeBSD, ошибка:

`morph_dict/agramtab/agramtab_.cpp:292:92: error: cannot pass object of non-trivial type 'std::string' (aka 'basic_string<char, char_traits<char>, allocator<char>>') through variadic function; call will abort at runtime [-Wnon-pod-varargs]
            throw CExpc(Format("line %s in  %s contains a dublicate gramcode",key.c_str(), m_InputJsonPath));`

--- a/agramtab/agramtab_.cpp
+++ b/agramtab/agramtab_.cpp
@@ -289,7 +289,7 @@ nlohmann::json CAgramtab::ReadFromFolder(std::string folder) {
         pAgramtabLine->m_PartOfSpeech = pos;
         size_t gram_index = GramcodeToLineIndex(gramcode.c_str());
         if (GetLine(gram_index)) {
-            throw CExpc(Format("line %s in  %s contains a dublicate gramcode",key.c_str(), m_InputJsonPath));
+            throw CExpc(Format("line %s in  %s contains a dublicate gramcode",key.c_str(), m_InputJsonPath.c_str()));
         }
         GetLine(gram_index) = pAgramtabLine;
         line_no++;

